client/app/components/compose.coffee

```diff
@@ -191,10 +189,7 @@ module.exports = Compose = React.createClass
                                 disable: if @state.saving then true else null
                                 type: 'button', onClick: @onDraft,
                                     if @state.saving
-                                        span null,
-                                            img
-                                                src: 'images/spinner-white.svg'
-                                                className: 'button-spinner'
+                                        span null, Spinner(white: true)
```

Comment:
I don't know, I just replaced what existed with the spinner component. Didn't rework the code around it. The span could simply be removed imo, but haven't digged into it.